packages/yourloops/lib/data/patient.ts

```diff
@@ -46,6 +46,7 @@ interface PatientProfile {
   lastName?: string;
   email: string;
   sex?: string;
+  referringDoctor?: string;
```

Comment:
I suspect that this values is never set when loading the patients

Reply:
indeed, the mapping was missing and the default value set